stdlib/platform/java/script.wyv

```diff
@@ -0,0 +1,4 @@
+require java
+import fileSystemClient
+val client = fileSystemClient(java)
```

Comment:
As mentioned above, this code should change to pass in the file system object, instead of `java`:
```
import fileSystem
val fileSys = fileSystem(java)
val client = fileSystemClient(fileSys)
```